components/breadcrumb/style/index.ts

```diff
@@ -67,11 +68,11 @@ const genBreadcrumbStyle: GenerateStyle<BreadcrumbToken, CSSObject> = (token) =>
       a: {
         color: token.linkColor,
         transition: `color ${token.motionDurationMid}`,
-        padding: `0 ${token.paddingXXS}px`,
+        padding: `0 ${unit(token.paddingXXS)}`,
         borderRadius: token.borderRadiusSM,
-        height: token.lineHeight * token.fontSize,
+        height: calc(token.lineHeight).mul(token.fontSize).equal(),
```

Comment:
```suggestion
        height: fontHeight,
```
所有 lineHeight * fontSize 都可以用 fontHeight 代替了